concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -24,6 +24,7 @@ params:
   GOVUK_NOTIFY_NO_SPL_MATCH_EMAIL_TEMPLATE_ID:
   GOVUK_NOTIFY_NO_SPL_MATCH_SMS_TEMPLATE_ID:
   GOVUK_NOTIFY_NO_SPL_MATCH_LETTER_TEMPLATE_ID:
+  METRICS_BASIC_AUTH: false
```

Comment:
it also needs to be set further down in the actual environment that paas will run